audiophiler/__init__.py

```diff
@@ -228,3 +231,11 @@ def get_harold_list(uid):
             harolds.append(harold.file_hash)
 
     return harolds
+
+def get_random_harold_list():
+    harold_list = Harold.query.all()
+    harolds = []
+    for harold in harold_list:
+        harolds.append(harold.file_hash)
```

Comment:
this is going to be a very slow process. How about we use sqlalchemy to select a random harold from the harolds table directly? Or choose a pre-determined file.